### DISCUSSION TITLE
Writing Olivetti Prodest PC1 disks?

### DISCUSSION THREAD

**Discussion by ZioGuido** [keirf/greaseweazle#302] [Q&A] [answered]
Hello everyone.
I'm trying to write some Olivetti PC1 disks downloaded from this link:
https://www.ti99iuc.it/web/?pageid=articoli_tech&artid=134

The files are in .DSK format which appears to be compatible with the GW, but when I load them I get a "Bad Signature" error.
I've tried converting them to .IMA using WinImage and then write them as either PC98.2d or atarist.720 or ibm.729 but the disks don't boot. With ibm.720 sometimes the writing fails around sector 50.

Any help? Thanks.

**Comment by keirf** [comment, marked as answer]
Greaseweazle interprets DSK files as CPC DSK image format. However, many systems actually use DSK for plain sector image files (which GW calls IMA or IMG). So first you should rename the file to have the `.img` suffix. And then you will need to know the sector layout. Probably it is `ibm.720`. So:
```
gw write --format=ibm.720 my_image.img
```
It's possible that the format isn't `ibm.720` though? To be sure you would need a raw dump of a disk formatted on the PC1.

**Comment by ZioGuido** [comment]
That's a very good advice. I have renamed the file to .IMA and wrote it using --format=atari.720 and it worked.

Thanks!

**Reply by keirf** [reply]
`ibm.720` is probably better, but `atari.720` is quite similar just with some sector skew applied.
